contents/docs/integrate/user-properties.mdx

```diff
@@ -68,17 +76,17 @@ Important notice: here with equal timestamps and previous method (rows 4 and 11)
 
 For `identify` this series of calls would mean we end up with `location=New York` and `browser=Safari`.
 ```js
-posthog.set('Alice', {'location': 'Rome'}, timestamp=1)
-posthog.set_once('Alice', '{browser': 'Chrome', timestamp=1)
-posthog.identify('Alice', {'$set': {'location': 'New York'}, '$set_once':{'browser':'Safari'}}, timestamp=1)
+posthog.set('Alice', {location: 'Rome'}, timestamp=1)
```

Comment:
this is confusing as it's not a valid call (the `timestamp` stuff that was here before)

Reply:
you mean I need to merge on top of the other PR. Or that I should change the timestamp to be a date from now'ish date time?

Reply:
Nah I just mean that the larger point here is that these _look_ like valid calls, but are not.

Not sure what the best way to represent them is

Reply:
aah I see. I can make them valid Python calls